trymito.io/components/Footer/Footer.tsx

```diff
@@ -78,7 +81,7 @@ const Footer = (): JSX.Element => {
                         <a href='https://www.youtube.com/channel/UCN9o_0m1fwCjigfIpnKr0oA/videos' target="_blank" rel="noreferrer">YouTube</a>
                     </li>
                     <li className='text-nav'>
-                        <a href='https://discord.gg/XdJSZyejJU' target="_blank" rel="noreferrer">Contact</a>
+                        <a href='[Slack](https://join.slack.com/t/trymito/shared_invite/zt-1h6t163v7-xLPudO7pjQNKccXz7h7GSg)' target="_blank" rel="noreferrer">Contact</a>
```

Comment:
I feel like this would be a good spot to have both of them for sure!

Reply:
Its already in the sitemap. There is a button for Discord and Slack.